vendor/github.com/hashicorp/go-plugin/client.go

```diff
@@ -780,7 +780,10 @@ func (c *Client) reattach() (net.Addr, error) {
 	// Verify the process still exists. If not, then it is an error
 	p, err := os.FindProcess(c.config.Reattach.Pid)
 	if err != nil {
-		return nil, err
+		// On Unix systems, FindProcess never returns an error.
```

Comment:
Could you please open this diff on [hashicorp/go-plugin](https://github.com/hashicorp/go-plugin)

Reply:
It is already there:
https://github.com/hashicorp/go-plugin/pull/125